.github/workflows/konflux-policy.yaml

```
@@ -73,11 +88,78 @@ jobs:
+        if git diff --quiet; then
+          echo "No changes to task definitions"
+          exit 0
+        fi
+
+        git config user.email "github-actions[bot]@users.noreply.github.com"
+        git config user.name "github-actions[bot]"
+
+        git add tasks/
+        git commit -m "Update task definitions from tested bundle
+
+${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)